demo/launch_demo_bmo.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Run the baremetal operator locally in "demo" mode.
+#
+# This script assumes that you have ~/.kube/config set up, which may
+# mean copying ocp/auth/kubeconfig after running the scripts to build
+# a dev environment.
+
+set -xe
+
+eval $(go env)
```

Comment:
As well as eval-ing the go env, we need to set the KUBECONFIG for `oc` below, either via common.sh:

```
SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
source ${SCRIPTDIR}/../common.sh
```

Or via a similar method that sets only KUBECONFIG to `$SCRIPTDIR/../ocp/auth/kubeconfig`?